### PR TITLE
映射文档更新

### PR DESCRIPTION
**Pull request by Xuxuanang**
torch.Tensor.baddbmm
torch.Tensor.baddbmm_
torch.Tensor.lu_solve
torch.Tensor.positive
torch.optim.lr_scheduler.LRScheduler
torch.special.gammaln
torch._foreach_lgamma
torch._foreach_lgamma_
torch._foreach_sqrt
torch._foreach_sqrt_
torch.hub.load_state_dict_from_url
torch.positive

### CODE REVIEW

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.lu_solve.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+### [torch.Tensor.lu_solve](https://pytorch.org/docs/stable/generated/torch.Tensor.lu_solve.html#torch-tensor-lu-solve)
+
+```python
+torch.Tensor.lu_solve(LU_data, LU_pivots)
+```
+
+### [paddle.linalg.lu_solve](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/linalg/lu_solve_cn.html)
+
+```python
+paddle.linalg.lu_solve(b, lu, pivots, trans="N", name=None)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.lu_solve.md:12]: 有`paddle.Tensor.lu_solve`吗

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.positive.md [hunk 2]
```diff
@@ -1,27 +1,15 @@
...
-
-# Paddle 写法
-def positive(x):
-    if x.dtype != paddle.bool:
-        return x
-    else:
-        raise RuntimeError("boolean tensors is not supported.")
-
-positive(x)
+paddle.positive(x)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.positive.md:12]: 有paddle.Tensor.positive吗

File: docs/guides/model_convert/convert_from_pytorch/api_difference/optimizer/torch.optim.lr_scheduler.LRScheduler.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+| optimizer     | learning_rate       | PyTorch 的 optimizer 类型是 torch.optim.Optimizer，Paddle 的 learning_rate 类型是 float，两者功能上不直接一致，但可通过设置 leaning_rate = optimizer.get_lr() 来对应一致。  |
+| last_epoch     | last_epoch       | 上一轮的轮数，重启训练时设置为上一轮的 epoch 数。参数完全一致。       |
+| -     | verbose       | 如果是 True，则在每一轮更新时在标准输出 stdout 输出一条信息。|
+
+### 转写示例
+
+```python
+# PyTorch 写法
+linear = torch.nn.Linear(10, 10)
+class Newscheduler(torch.optim.lr_scheduler.LRScheduler):
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/optimizer/torch.optim.lr_scheduler.LRScheduler.md]: 命名 CustomScheduler 吧

File: docs/guides/model_convert/convert_from_pytorch/api_difference/others/torch.special.gammaln.md [hunk 4]
```diff
@@ -6,10 +6,10 @@
 torch.special.gammaln(input, *, out=None)
 ```
 
-### [paddle.lgamma](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/lgamma_cn.html#lgamma)
+### [paddle.gammaln](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/gammaln_cn.html#gammaln)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/others/torch.special.gammaln.md:9]: paddle.lgamma是其他API的别名吗，这个对应到torch哪个，这几个gamma api需要细致对比下

File: docs/guides/model_convert/convert_from_pytorch/api_difference/torch/torch._foreach_sqrt_.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+torch._foreach_sqrt_(self)
+```
+
+Paddle 无此 API，需要组合实现。
+
+### 转写示例
+
+```python
+# PyTorch 写法
+torch._foreach_sqrt_(tensors)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/torch/torch._foreach_sqrt_.md:15]: paddle有paddle.sqrt_吗

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.lu_solve.md (hunk 1 above)
Comment (Xuxuanang) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.lu_solve.md:12]: 没有

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.positive.md (hunk 2 above)
Comment (Xuxuanang) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.positive.md:12]: 没有

File: docs/guides/model_convert/convert_from_pytorch/api_difference/optimizer/torch.optim.lr_scheduler.LRScheduler.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+    ...
+
+sgd = torch.optimizer.SGD(lr=0.5, parameters=linear.parameters())
+scheduler = Newscheduler(optimizer=sgd, lr_lambda=lambda x:0.95**x)
+
+# Paddle 写法
+linear = paddle.nn.linear(10, 10)
+class Newscheduler( paddle.optimizer.lr.LRScheduler):
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/optimizer/torch.optim.lr_scheduler.LRScheduler.md]: 空格不需要

File: docs/guides/model_convert/convert_from_pytorch/api_difference/optimizer/torch.optim.lr_scheduler.LRScheduler.md (hunk 3 above)
Comment (zhwesky2010) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/optimizer/torch.optim.lr_scheduler.LRScheduler.md]: 示例代码再详细点，把\__init\__的签名也加进来，说明两者基类不同，持有的关系是不同的。

File: docs/guides/model_convert/convert_from_pytorch/api_difference/torch/torch._foreach_sqrt_.md (hunk 5 above)
Comment (Xuxuanang) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/torch/torch._foreach_sqrt_.md:15]: 没有

File: docs/guides/model_convert/convert_from_pytorch/api_difference/others/torch.special.gammaln.md (hunk 4 above)
Comment (Xuxuanang) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/others/torch.special.gammaln.md:9]: 是别名 或者说这两个其实是等价的 不过torch没有标出来 paddle.lgamma对应到torch.lgamma上面了

Comment (zhwesky2010) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/others/torch.special.gammaln.md:9]: infoflow 2025-06-13 18-01-45

这两个好像实现不同，是别名吗

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.lu_solve.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+```
+
+Pytorch 为 Tensor 类方法，Paddle 为普通函数，另外 Paddle 相比 PyTorch 支持更多其他参数，具体如下：
+
+### 参数映射
+
+| PyTorch       | PaddlePaddle | 备注                                                   |
+| ------------- | ------------ | ------------------------------------------------------ |
+| LU_data  |   lu   | 表示 LU 分解结果矩阵，由 L、U 拼接组成，仅参数名不一致。 |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.lu_solve.md:22]: 上面再加一行，self与b对应吧

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.positive.md [hunk 8]
```diff
@@ -1,27 +1,15 @@
...
 ```
+
+两者功能一致，无参数。
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.positive.md]: torch是类方法，paddle不是类方法，参考一下其他的描述

**Review by zhwesky2010 — APPROVED** [review]
LGTM
